manifest.json

```diff
@@ -716,6 +716,24 @@
       "useCaseName": "Multi-step workflow"
     }
   ],
+  "rust": [
+    {
+      "directory": "rust/cookiecutter-aws-sam-rust-hello",
+      "displayName": "Hello World Example",
+      "dependencyManager": "cargo",
+      "appTemplate": "hello-world",
+      "packageType": "Zip",
+      "useCaseName": "Serverless API"
+    },
+    {
+      "directory": "rust/cookiecutter-aws-sam-rust-hello-dynamodb",
+      "displayName": "Hello World Example with DynamoDB",
+      "dependencyManager": "cargo",
+      "appTemplate": "hello-world",
+      "packageType": "Zip",
+      "useCaseName": "Serverless API"
```

Comment:
Can we give another use case name for this? Is that fitting any other templates that we have?

Reply:
I don't see any that would match, but I think there's some benefits to having an example that shows how to reuse and mock the AWS SDK client within a Lambda function.

Should I use "Hello World with DynamoDB" as a name? What are the consequences of creating new useCaseNames?